types/src/crypto/hpke/sqlx_impls.rs

```diff
@@ -0,0 +1,129 @@
+// SPDX-FileCopyrightText: 2025 Phoenix R&D GmbH <hello@phnx.im>
+//
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+use crate::crypto::secrets::SecretBytes;
+
+use super::{ClientIdKeyType, DecryptionKey, EncryptionKey};
+
+impl<'q, KT, DB: ::sqlx::Database> ::sqlx::encode::Encode<'q, DB> for EncryptionKey<KT>
```

Comment:
Minor: maybe we could import sqlx types to reduce a little bit the syntax noise? Since it is a dedicated file, there should not be conflicts and confusions without other type names.